src/applications/mhv-landing-page/components/Welcome.jsx

```diff
@@ -26,31 +27,35 @@ const Welcome = ({ loading, name }) => (
       )}
       {!name && <>Welcome</>}
     </h2>
-    <div className="vads-u-font-size--md medium-screen:vads-u-font-size--lg vads-u-display--flex vads-u-align-items--center">
+    <div className="vads-u-display--flex vads-u-align-items--center">
       <span
         className={classnames(
-          'vads-u-color--primary-dark',
           'vads-u-padding-left--4',
           'vads-u-padding-right--0p5',
+          'vads-u-color--primary-dark',
         )}
       >
         <va-icon icon="account_circle" size={3} />
       </span>
-      <va-link
+      <a
+        className="vads-u-font-size--md medium-screen:vads-u-font-size--lg"
         href="/profile"
-        text="Profile"
-        className="vads-u-visibility--screen-reader"
-      />
+        onClick={() => {
+          datadogRum.addAction('Click on Landing Page: Welcome - Profile');
+          recordEvent({
+            event: 'nav-linkslist',
```

Comment:
I think we'd want to record this as something other than a `nav-linkslist` since that's not what this is. I'd hope we could use [attributes like on va-link](https://github.com/department-of-veterans-affairs/component-library/blob/8d611eb2037ae1b63ff6315110d3c0da9bbc7381/packages/web-components/src/components/va-link/va-link.tsx#L96-L102)…

Reply:
@dcloud - it's ready for another check.
I used `nav-linkslist` initially to keep the GA events the same as other card links on the landing page. But I agree with your suggestion to use event like va-link as it better represent a nav link event.
I've updated it with attributes like on va-link, except for event-source attribute which I left out as va-link sets it specifically for component-library `event-source: "component-library"`